Mail/Views/Thread/WebView/WebViewModel.swift

```diff
@@ -48,7 +48,12 @@ final class WebViewModel: NSObject, ObservableObject {
     }
 
     override init() {
-        webView = WKWebView()
+        var configuration = WKWebViewConfiguration()
```

Comment:
```suggestion
        let configuration = WKWebViewConfiguration()
```
Can it be a let as WKWebViewConfiguration is a class?

Reply:
Yup, i thought it was a a struct